it/it-tests/src/test/java/it/qualityProfile/QualityProfilesBuiltInTest.java

```diff
@@ -21,59 +21,116 @@
 
 import com.sonar.orchestrator.Orchestrator;
 import it.Category6Suite;
+import java.util.function.Predicate;
+import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
+import org.sonarqube.ws.QualityProfiles;
 import org.sonarqube.ws.QualityProfiles.SearchWsResponse;
 import org.sonarqube.ws.QualityProfiles.SearchWsResponse.QualityProfile;
+import org.sonarqube.ws.client.HttpException;
 import org.sonarqube.ws.client.WsClient;
 import org.sonarqube.ws.client.organization.CreateWsRequest;
+import org.sonarqube.ws.client.qualityprofile.ActivateRuleWsRequest;
+import org.sonarqube.ws.client.qualityprofile.CopyRequest;
+import org.sonarqube.ws.client.qualityprofile.DeleteRequest;
 import org.sonarqube.ws.client.qualityprofile.SearchWsRequest;
+import org.sonarqube.ws.client.qualityprofile.SetDefaultRequest;
 
 import static it.Category6Suite.enableOrganizationsSupport;
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+import static org.apache.commons.lang.RandomStringUtils.randomAscii;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.tuple;
 import static util.ItUtils.newAdminWsClient;
 import static util.ItUtils.newOrganizationKey;
 
 public class QualityProfilesBuiltInTest {
 
-  private static final String ANOTHER_ORGANIZATION = newOrganizationKey();
+  private static final String ORGANIZATION = newOrganizationKey();
+  private static final String RULE_ONE_BUG_PER_LINE = "xoo:OneBugIssuePerLine";
+
   @ClassRule
   public static Orchestrator orchestrator = Category6Suite.ORCHESTRATOR;
+
   private static WsClient adminWsClient;
 
   @BeforeClass
   public static void setUp() {
     enableOrganizationsSupport();
     adminWsClient = newAdminWsClient(orchestrator);
+    adminWsClient.organizations().create(new CreateWsRequest.Builder()
+      .setKey(ORGANIZATION)
+      .setName(ORGANIZATION).build());
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    adminWsClient.organizations().delete(ORGANIZATION);
   }
 
   @Test
-  public void xoo_profiles_provided() {
-    SearchWsResponse result = adminWsClient.qualityProfiles().search(new SearchWsRequest());
+  public void xoo_profiles_provided_copied_to_new_organization() {
```

Comment:
the title could be more explicit : `built_in_profiles_are_associated_to_new_organizations`.